pom.xml

```
@@ -72,65 +72,126 @@
         <tag>${scmTag}</tag>
     </scm>
 
-    <dependencies>
-    </dependencies>
+    <profiles>
+        <profile>
+            <id>linux</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

```suggestion
                <os>
                    <family>linux</family>
                </os>
```

or what am I missing?

---

my mistake, i meant to remove the os family restriction.

---

removed os family restriction, changed `id` name to avoid confusion
ef79fc334efeb8b4db5926e0954aba524bff2b53